daemon/logger/journald/journald.go

```diff
@@ -48,10 +49,17 @@ func New(ctx logger.Context) (logger.Logger, error) {
 		name = name[1:]
 	}
 
+	// parse log tag
+	tag, err := loggerutils.ParseLogTag(ctx, "")
```

Comment:
Let me know when you make one. I wonder if we even need second argument if it's always the same for all loggers.

```
daemon/logger/fluentd/fluentd.go:	tag, err := loggerutils.ParseLogTag(ctx, "docker.{{.ID}}")
daemon/logger/gelf/gelf.go:	tag, err := loggerutils.ParseLogTag(ctx, "")
daemon/logger/journald/journald.go:	tag, err := loggerutils.ParseLogTag(ctx, "")
daemon/logger/splunk/splunk.go:	tag, err := loggerutils.ParseLogTag(ctx, "{{.ID}}")
daemon/logger/syslog/syslog.go:	tag, err := loggerutils.ParseLogTag(ctx, "{{.ID}}")
```